test/test_tensorexpr.py

```diff
@@ -1210,19 +1208,6 @@ def run_where(x, y):
         y = run_where(a, b)
         np.testing.assert_allclose(x.numpy(), y.numpy())
 
-    @unittest.skipIf(not torch.cuda.is_available(), "requires CUDA")
-    @unittest.skip("temporarily disable")
-    def test_unused(self):
```

Comment:
I don't understand what this test was checking and it seems that we have other tests for `rand_like`, so let's just remove this one (it works, but I think it's redundant).